backend/app/views/spree/admin/tax_rates/index.html.erb

```diff
@@ -42,7 +42,13 @@
       <tr id="<%= spree_dom_id tax_rate %>" data-hook="rate_row" class="<%= cycle('odd', 'even')%>">
         <td class="align-center"><%=tax_rate.zone.try(:name) || Spree.t(:not_available) %></td>
         <td class="align-center"><%=tax_rate.name %></td>
-        <td class="align-center"><%=tax_rate.tax_category.try(:name) || Spree.t(:not_available) %></td>
+        <td class="align-center">
+          <% if tax_rate.tax_categories.any? %>
+            <%= tax_rate.tax_categories.map(&:name).join(", ") %>
```

Comment:
Here, you might want to use an internationalized separator, as not all locales will work with a comma separated list. Most will though, so I won't insist. :)